src/showplanner/PISModal.tsx

```diff
@@ -46,16 +46,8 @@ function PersistNotice() {
       <div className="p-2 alert-primary">
         <h2>Welcome to WebStudio {process.env.REACT_APP_VERSION}!</h2>
         <p>
-          As of this version,{" "}
```

Comment:
Should we just get rid of `PersistNotice` entirely?

Reply:
Maybe!